inginious/frontend/plugins/problem_bank/react_app/src/bank_course_list.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import { Well, Row, Col } from 'react-bootstrap';
+import './index.css';
+
+import UltimatePagination from './ultimate_pagination';
+import BankCourse from './bank_course'
+import CourseAutosuggest from './course_autosuggest'
+import CustomAlert from './custom_alert';
+
+class BankCourseList extends React.Component {
+
+    render() {
+        let courses = this.props.courses.map((course, i) => {
+            if(i >= ((this.props.page - 1) * this.props.limit) && i < (this.props.page * this.props.limit)) {
```

Comment:
Try to make if statements read as English statements.

So you can make an explanatory variable such as ```Is in bounds of page``` or something like that to explaing the intent of the computation and comparison you are making